requirements.txt

```diff
@@ -34,6 +34,8 @@ click==8.1.3
     #   flask
 clickclick==20.10.2
     # via connexion
+colorama==0.4.6
+    # via click
```

Comment:
what is colorama being used for?

Reply:
Unsure, was autogenerated but will be removed

Reply:
removed and file matches that of master